niworkflows/anat/ants.py

```diff
@@ -951,3 +1071,47 @@ def _conform_mask(in_mask, in_reference):
         np.asanyarray(nii.dataobj).astype("int16"), ref.affine, hdr
     ).to_filename(out_file)
     return out_file
+
+
+def _matchlen(value, reference):
+    return [value] * len(reference)
+
+
+def _imsum(op1, op2, out_file=None):
+    import nibabel as nb
+
+    im1 = nb.load(op1)
+
+    data = im1.get_fdata() + nb.load(op2).get_fdata()
```

Comment:
is 64bit precision necessary here?

Reply:
I would reverse the question and say, do we need to cast back to float32? after all these are 3D maps.